docs/1.0/manual/microservice/dashboard-advanced.md

```diff
@@ -59,36 +61,43 @@ LIMIT 100;
 * etc.
 
 ### 类型转换
-* int(value) 将value转化为int64，不符合限定类型返回error
+* int(value) 将value转化为int64，不符合限定类型返回error<br>
 	   value type:bool、int、uint、float、string、time.Duration、time.Time
-	   <br>
   
-* bool(value) 将value转化为bool，不符合限定类型返回error
+* bool(value) 将value转化为bool，不符合限定类型返回error<br>
        value type:bool、int、uint、float、string、time.Duration、time.Time
-	   <br>
 	   
-* float(value) 将value转化为float，不符合限定类型返回error
+* float(value) 将value转化为float，不符合限定类型返回error<br>
        value type:bool、int、uint、float、string、time.Duration、time.Time
-	   <br>
 	   
-* string(value) 将value转化为string，转换失败返回错误
+* string(value) 将value转化为string，转换失败返回错误<br>
        value type:interface{}
-	   <br>
 	   
-* duration(value) 将value转化为time.Duration，不符合限定类型返回error
+* duration(value) 将value转化为time.Duration，不符合限定类型返回error<br>
        value type:bool、int、uint、float、string、time.Duration
-	   <br>
 
 * etc.
 
-### 其他函数
-* if(cond, true_expression, false_exprission), 等价于 cond ? true_expression : false_exprission
-* include(key, value1, value2...), 等价于 key IN (value1, value2), 返回 boolean
+### 关系运算函数
 * eq(value1, value2), 等价于 value1 == value2，返回boolean
+* neq(value1, value2), 等价于 value1 != value2，返回boolean
 * gt(value1, value2), 等价于 value1 > value2，返回boolean
 * gte(value1, value2), 等价于 value1 >= value2，返回boolean
 * lt(value1, value2), 等价于 value1 < value2，返回boolean
 * lte(value1, value2), 等价于 value1 <= value2，返回boolean
+
+### 条件运算函数
+* if(cond, true_expression, false_exprission), 等价于 cond ? true_expression : false_exprission
```

Comment:
typo `false_exprission `

Reply:
> typo `false_exprission `

if condition was false , case the false exprission

Reply:
is `expression`, not `exprission`